compiler/rustc_codegen_llvm/src/lib.rs

```diff
@@ -401,7 +403,7 @@ impl CodegenBackend for LlvmCodegenBackend {
 pub struct ModuleLlvm {
     llcx: &'static mut llvm::Context,
     llmod_raw: *const llvm::Module,
-    tm: &'static mut llvm::TargetMachine,
+    tm: TargetMachineWrapper,
```

Comment:
From looking at it, it doesn't seem like the target machine borrows from the LLVM context or module, but it would be nice to leave a comment here saying that the machine is dropped by `TargetMachineWrapper` and that it doesn't borrow from the pointers above (after you've confirmed that it doesn't).